website/src/content/api-reference/commercetools-frontend-actions-global.mdx

```diff
@@ -0,0 +1,240 @@
+| `text`              | `string`   |    -     | -                                                                                     | -       | The text content of the notification.              |
+
+<Info>
+
+For defining the domain and the kind of a notification we recommend using the values exposed from `@commercetools-frontend/constants`.
+
+For instance:
+
+```js
+import { 
+  DOMAINS, 
+  NOTIFICATION_KINDS_SIDE
+} from '@commercetools-frontend/constants';
+
+const kind = NOTIFICATION_KINDS_SIDE.success;
+const domain = DOMAINS.SIDE;
+```
+
+</Info>
+
+- `meta` object properties
+
+| Property            | Type       | Required | Values         | Default                                           | Description                                                                            |
+| ------------------- | ---------- | :------: | -------------- | ------------------------------------------------- | -------------------------------------------------------------------------------------- |
+| `dismissAfter`      | `number`   |     -    | -              | `5000` (only for the `success` notification)      | Delay (in milliseconds) between the appearing and the closing of the notification.     |
+| `onDismiss`         | `function` |     -    | -              | -                                                 | Called when the notification closes. Receives notification `id` as an argument.        |
+
+
+## showApiErrorNotification
+
+An action creator to dispatch an (page) error notification for [API errors](https://docs.commercetools.com/api/errors) of the commercetools platform API.
+Takes as an argument an object with `errors` property being an error object or an array of error objects.
+
+Some sources or errors (for instance GraphQL APIs) return an array or object. `showApiErrorNotification` action creator accepts either as the value of `errors` property. 
+
+```js
+const errors = [
+  {
+    message:
+      "A duplicate value '\"Test channel\"' exists for field 'key'.",
+    code: "DuplicateField",
+    extensions: {
+      code: "DuplicateField";
+    };
+  },
+];
+// ...
+dispatch(showApiErrorNotification({ errors })); // dispatch method from Redux store
+```
+
+## handleActionError
+
+An action creator that attempts to handle a generic error and dispatch the appropriate (page) notification error if possible. Use this if you simply need to ensure a notification error is shown without knowing what the exact error is.
+
+Takes an error object as an argument:
+
+- API error - an error object containing `statusCode` and `body` object
+```js
+import { STATUS_CODES } from '@commercetools-frontend/constants';
+const error = {
+  statusCode: STATUS_CODES.FORBIDDEN, // statusCode: 403
+  body: {
+    message: "This is an error",
+    errors: [
+      {
+        message:
+          "This is an error",
+        code: "Invalid",
+      },
+    ],
+  },
+}
+// ...
+dispatch(handleActionError(error)); // dispatch method from Redux store
+```
```

Comment:
I think it would he helpful to explain which property is used for the notification error.
For instance, I'm wondering whether `error.body.message` or a concatenation of `error.body.errors[n].message`.